FirebaseRemoteConfig/Sources/RCNDevice.m

```diff
@@ -60,132 +60,10 @@ RCNDeviceModel FIRRemoteConfigDeviceSubtype(void) {
   return [[[NSLocale currentLocale] objectForKey:NSLocaleCountryCode] lowercaseString];
 }
 
-NSDictionary<NSString *, NSArray *> *FIRRemoteConfigFirebaseLocaleMap(void) {
-  return @{
-    // Albanian
-    @"sq" : @[ @"sq_AL" ],
-    // Belarusian
-    @"be" : @[ @"be_BY" ],
-    // Bulgarian
-    @"bg" : @[ @"bg_BG" ],
-    // Catalan
-    @"ca" : @[ @"ca", @"ca_ES" ],
-    // Croatian
-    @"hr" : @[ @"hr", @"hr_HR" ],
-    // Czech
-    @"cs" : @[ @"cs", @"cs_CZ" ],
-    // Danish
-    @"da" : @[ @"da", @"da_DK" ],
-    // Estonian
-    @"et" : @[ @"et_EE" ],
-    // Finnish
-    @"fi" : @[ @"fi", @"fi_FI" ],
-    // Hebrew
-    @"he" : @[ @"he", @"iw_IL" ],
-    // Hindi
-    @"hi" : @[ @"hi_IN" ],
-    // Hungarian
-    @"hu" : @[ @"hu", @"hu_HU" ],
-    // Icelandic
-    @"is" : @[ @"is_IS" ],
-    // Indonesian
-    @"id" : @[ @"id", @"in_ID", @"id_ID" ],
-    // Irish
-    @"ga" : @[ @"ga_IE" ],
-    // Korean
-    @"ko" : @[ @"ko", @"ko_KR", @"ko-KR" ],
-    // Latvian
-    @"lv" : @[ @"lv_LV" ],
-    // Lithuanian
-    @"lt" : @[ @"lt_LT" ],
-    // Macedonian
-    @"mk" : @[ @"mk_MK" ],
-    // Malay
-    @"ms" : @[ @"ms_MY" ],
-    // Maltese
-    @"mt" : @[ @"mt_MT" ],
-    // Polish
-    @"pl" : @[ @"pl", @"pl_PL", @"pl-PL" ],
-    // Romanian
-    @"ro" : @[ @"ro", @"ro_RO" ],
-    // Russian
-    @"ru" : @[ @"ru_RU", @"ru", @"ru_BY", @"ru_KZ", @"ru-RU" ],
-    // Slovak
-    @"sk" : @[ @"sk", @"sk_SK" ],
-    // Slovenian
-    @"sl" : @[ @"sl_SI" ],
-    // Swedish
-    @"sv" : @[ @"sv", @"sv_SE", @"sv-SE" ],
-    // Turkish
-    @"tr" : @[ @"tr", @"tr-TR", @"tr_TR" ],
-    // Ukrainian
-    @"uk" : @[ @"uk", @"uk_UA" ],
-    // Vietnamese
-    @"vi" : @[ @"vi", @"vi_VN" ],
-    // The following are groups of locales or locales that sub-divide a
-    // language).
-    // Arabic
-    @"ar" : @[
-      @"ar",    @"ar_DZ", @"ar_BH", @"ar_EG", @"ar_IQ", @"ar_JO", @"ar_KW",
-      @"ar_LB", @"ar_LY", @"ar_MA", @"ar_OM", @"ar_QA", @"ar_SA", @"ar_SD",
-      @"ar_SY", @"ar_TN", @"ar_AE", @"ar_YE", @"ar_GB", @"ar-IQ", @"ar_US"
-    ],
-    // Simplified Chinese
-    @"zh_Hans" : @[ @"zh_CN", @"zh_SG", @"zh-Hans" ],
-    // Traditional Chinese
-    // Remove zh_HK until console added to the list. Otherwise client sends
```

Comment:
I don't have enough context on this to know if it's still relevant, but I'd prefer if this change doesn't accidentally cause unexpected behavior for existing locale targeting

could you just add the additional mappings for now instead of replacing the whole map?

Reply:
Thanks @karenyz. Done updating the list.

Reply:
thank you!